masterlist.yaml

```diff
@@ -255,18 +255,18 @@ groups:
   - name: &coreGroup Core Mods
     after: [ *lowPriorityGroup ]
 
-  - name: &ocsGroup Open Cities
-    after: [ *coreGroup ]
-
   - name: &altStartGroup Alternate Start
-    after: [ *ocsGroup ]
+    after: [ *coreGroup ]
```

Comment:
Not sure what effect swapping group order will have on current users load order.
In hindsight it my be better to not have a group named after a mod, so that it isn't intrinsically linked.

Reply:
It's better to be conservative about making changes because of this, but at least the group names haven't changed, so load orders will change but no user metadata should break (apart from possibly introducing new cycles).